src/client/src/models/Employee.js

```diff
@@ -1,16 +1,27 @@
-export default class Employee {
-    constructor(id, firstName, lastName, email, roles) {
-        this.id=id;
-        this.firstName=firstName;
-        this.lastName=lastName;
-        this.email=email;
-        this.roles=roles;
-    }
+ export default class Employee {
```

Comment:
There is an extra space here before the export.

And for the indentation, maybe you misunderstood, but it needs to be a tab (and its size is 2). You can view the file taskwipe/src/client/src/index.js commited in this pull request for an example.
The sizing of the tab can be configured in your IDE/text editor.